pkgs/tools/misc/fd/default.nix

```diff
@@ -15,10 +15,17 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  preFixup = ''
+  # skip flaky test
+  checkFlags = [
+    "--skip=test_owner_current_group"
+  ];
+
+  postInstall = ''
     installManPage doc/fd.1
 
-    installShellCompletion $releaseDir/build/fd-find-*/out/fd.{bash,fish}
+    installShellCompletion --cmd fd \
+      --bash <($out/bin/fd --gen-completions bash) \
```

Comment:
https://github.com/sharkdp/fd/releases/tag/v8.5.0

Looks like this changed in 8.5.0 but it wasn't mentioned in the release notes.